src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/AbstractFunctional.java

```diff
@@ -66,18 +66,23 @@ public void beforeTestClass() {
     }
 
     protected static UserProfileCreationResponse createUserProfile(
-            UserProfileCreationData userProfileCreationData, HttpStatus httpStatus) throws Exception {
+            UserProfileCreationData userProfileCreationData, HttpStatus httpStatus, String params) throws Exception {
 
         UserProfileCreationResponse resource = testRequestHandler.sendPost(
                 userProfileCreationData,
                 httpStatus,
-                requestUri,
+                requestUri + params,
```

Comment:
instead of +, can we use append function?

Example:
requestUri.concat(params)

Reply:
Done !!! ☑️